vitest.config.ts

```diff
@@ -0,0 +1,39 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite'
```

Comment:
looks like this file is formatted differently from our usual "double quotes with semicolons"